### ISSUE TITLE
Get Gridsome to use a newer version of remark

### ISSUE THREAD

**Issue by NickSto** [galaxyproject/galaxy-hub#1004]
Gridsome is stuck on old versions of all the remark packages. `@gridsome/transformer-remark` uses `remark-parse` version 6.0.3 and `remark-html` version 8.0.0. And with the lack of Gridsome releases this year, it's time to see if there are other ways to address this.

This is the root cause of issues like #979 and #1003. If we were able to somehow get it to use newer versions of these packages, it'd fix those issues and/or remove the need for workarounds.

Looking at the source of transformer-remark, it looks like there's an option, `transformers.remark.stringifier`, which lets you replace their stringifier (they use `remark-html` by default).

**Comment by dannon** [comment]
@NickSto Have you looked into whether or not there are backwards compatibility issues with the new versions?  If not, I can try just pinning them to the latest.

**Comment by NickSto** [comment]
Oh I didn't know we could (or should) force a different version on a dependency of a dependency! Yeah then we could maybe define a bunch of tests and try it out?

**Comment by dannon** [comment]
Yep, we can totally do that.  I don't know if we *should*, hence the question about compatibility over version jumps ;)  I'll set up a branch showing how this could work.

**Comment by NickSto** [comment]
Looking into this more, I haven't been able to reproduce the escaping issues with the old versions of remark. I expect I'm not replicating Gridsome's specific remark configuration, since I can't imagine where else an incorrect serialization issue could be coming from.

The test I performed involved installing Gridsome's versions of these package in a separate project:
- unified: 7.1.0
- remark-html: 8.0.0
- remark-parse: 6.0.3

And then testing with this code:
```javascript
const unified = require('unified');
const remarkParse = require('remark-parse');
const remarkHtml = require('remark-html');
const processor = unified().use(remarkParse).use(remarkHtml);
processor.stringify(processor.parse('Q\&A'));
```
The result is the same HTML whether or not the backslash is included.
